test/inspector/test-on-off.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const common = require('../common');
+common.skipIfInspectorDisabled();
+const helper = require('./inspector-helper.js');
+
+function testResume(session) {
+  session.sendCommandsAndExpectClose([
+    { 'method': 'Runtime.runIfWaitingForDebugger' }
+  ]).expectStderrOutput('Debugger listening on');
+}
+
+function runTests(harness) {
+  harness
+    .runFrontendSession([
+      testResume,
+    ]).expectShutDown(0);
+}
+
+const script = 'process._debugEnd();' +
```

Comment:
Why the complex `script`?
Run once with just `_debugEnd` since this is the change.
We want to know if just this is failing, now it could fail on any of the two.
You can do a second run with this script.